src/pages/PublicAppointments/PatientRegistration.tsx

```diff
@@ -362,6 +362,19 @@ export function PatientRegistration(props: PatientRegistrationProps) {
                         <span className="text-xs text-gray-500">
                           {t("age_notice")}
                         </span>
+                        {form.getValues("year_of_birth") && (
+                          <div className="text-sm font-bold">
+                            {Number(form.getValues("year_of_birth")) <= 0 ? (
```

Comment:
Shouldn't this be CurrentYear - form(year_of_birth)

Reply:
Nah, "year_of_birth" was technically age (holdover from previous version, fixed/renamed it).